src/rgw/rgw_op.cc

```diff
@@ -2214,6 +2215,16 @@ void RGWGetObj::execute(optional_yield y)
   std::unique_ptr<RGWGetObj_Filter> run_lua;
   map<string, bufferlist>::iterator attr_iter;
 
+  bool rgw_perf_counters_cache = s->cct->_conf.get_val<bool>("rgw_perf_counters_cache");
```

Comment:
lets avoid this lookup-val-by-string version and rely on the legacy version `cct->_conf->rgw_perf_counters_cache` that doesn't require a mutex